### ISSUE TITLE
password protect submitted news

### ISSUE THREAD

**Issue by oui3kings** [e107inc/e107#3088]
Is it or would it be possible to add the feature for someone in submitnews to create a password to access the submission?

**Comment by Moc** [comment]
It's based on userclass now which should be sufficient I suppose?

**Comment by oui3kings** [comment]
True. I guess I was wondering if members/part of a particular userclass could add a password to their submission similar to how it can be done with custom pages.
